src/Sylius/Behat/Context/Transform/LexicalContext.php

```diff
@@ -18,12 +18,14 @@
 final class LexicalContext implements Context
 {
     /**
-     * @Transform /^"(\-)?(?:€|£|￥|\$)((?:\d+\.)?\d+)"$/
+     * @Transform /^"(\-)?(?:€|£|￥|\$)([^"]+)"$/
+     * @Transform /^"(\-)?(?:€|£|￥|\$)([^"]+\.[^"]+)"$/
```

Comment:
```suggestion
     * @Transform /^"(\-)?(?:€|£|￥|\$)([0-9,]+)"$/
     * @Transform /^"(\-)?(?:€|£|￥|\$)([0-9,]+\.[0-9]{2})"$/
```
If we want to be more specific and less greedy, thought about removing the first regex, but it's better for DX to leave it with the validation below than simply not match it without cents.

Reply:
I believed that leaving it looser would allow for incorrect prices to be passed into the validation part and directly informing the developer

Reply:
I do agree, but it still should be a price, using it for `$ABCD.EF` IMO makes no sense

Reply:
I combined these two regexes into one that only accepts digits but allows for any number of decimal digits to validate them in the helper function